config/config/common-config.yaml

```diff
@@ -175,7 +175,22 @@ fluentd:
       #   </match>
 
   aggregator:
-    buffer: {}
+    # All buffer settings are available, keys are transformed to snake_case as required by fluentd config
+    buffer:
+      chunkLimitSize: 50MB
+      totalLimitSize: 9GB
+
+      flushMode: interval
+      flushInterval: 5s
```

Comment:
So we can at most loose 5s worth of logs if fluentd were to crash?

Reply:
This is when it pushes to S3, so at most 5s if the PVC is lost, so this is way to low given how frequent it would push to S3.

Reply:
Set it to 15m now, it will still push when the chunk is read after ~10m, so this is in case that doesn't happen.

Reply:
But isn't flushing just for persisting the buffer on disk in our case?

Reply:
No the flushing here is done to the output.